BookEcommerce.Models/Entities/MailSettings.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookEcommerce.Models.Entities
+{
+    public class MailSettings
+    {
+        //public MailSettings(string? mail, string? displayName, string? password, string? host, int port)
+        //{
+        //    Mail = mail;
+        //    DisplayName = displayName;
+        //    Password = password;
+        //    Host = host;
+        //    Port = port;
+        //}
+
+        public string Mail { get => "huy2002109@gmail.com"; }
+        public string DisplayName { get => "BookCommerce"; }
+        public string Password { get => "askronkvelldgtke"; }
+        public string Host { get => "smtp.gmail.com"; }
+        public int Port { get => 587; }
```

Comment:
what is it used for?

Reply:
that is the config variable of the MailSender, but when i put it into the config file and get it out by using IConfiguration, it does not work and return all my variables as null so i just put it there to make sure my service could work well.